src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -1,5 +1,24 @@
-<div class="detail-list-container" [class.containerPadd-nologin] = "!loggedIn" >
-  
+<div *ngIf="loggedIn && showTypesModal" class="types-modal">  
+  <h5>Click to select a type</h5>
+  <ul>
+    <li *ngFor="let type of workItemTypes" (click)="onChangeType(type.name)">
+      <a>
```

Comment:
Don't use a tag if no need of href attribute. Its behavior may change in different browser.